conf.py

```diff
@@ -215,8 +220,13 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'pythondevelopersguide', u"Python Developer's Guide Documentation",
-     [u'Brett Cannon'], 1)
+    (
+        'index',
+        'pythondevelopersguide',
+        u"Python Developer's Guide Documentation",
+        [u'Brett Cannon'],
+        1,
+    ),
 ]
```

Comment:
Any reason to keep the ``u`` prefix around, especially since it's only used on some of the strings?
I would also put the square brackets and parentheses on the same line (i.e. ``[(`` and ``)]``) and remove a level of indentation.

Reply:
Black doesn't hug multiple levels of brackets to make it trivial to scan logical nesting levels visually.

Reply:
The main issue is the fact that 3 lines are "wasted" for parentheses/brackets.  The argument is also similar to the one I made below.  I would either use:
```python
man_pages = [('index', 'pythondevelopersguide',
              u"Python Developer's Guide Documentation",
              [u'Brett Cannon'], 1)]
```
This takes 1/3 of vertical space compared to the "expanded" version (but I would only use it in case of a single tuple) or
```python
man_pages = [
    ('index', 'pythondevelopersguide',
     u"Python Developer's Guide Documentation",
     [u'Brett Cannon'], 1),
]
```
This takes roughly half of the vertical space, and I would use it if I want/need more horizontal space or have (or planning to add) more tuples.

It also depends on the context: if it's a module-level constant I can splurge a bit and waste more vertical space, if it's a variable or a function call in the middle of a 10-lines function, I would avoid making the function twice as long if possible.  The fact that the style is context-dependent is also the reason why I generally avoid using tools that alter the formatting I used.